frontend/src/components/common/table/table.tsx

```diff
@@ -32,16 +34,23 @@ const Table: React.FC<Props> = ({ columns, data = [] }) => {
       <tbody {...getTableBodyProps()}>
         {rows.map((row, i) => {
           prepareRow(row);
-          return (
+          const episodeId = row.original.episodeId;
+          return (            
             <tr {...row.getRowProps()} key={i} className={styles.episodeRow}>
               {row.cells.map((cell, i) => {
-                return (
-                  <td {...cell.getCellProps()} key={i}>
+                return ( i === 1 
+                  ? <td {...cell.getCellProps()} key={i}>
+                    <Link to={`${AppRoute.EPISODES}/${episodeId}`} key={i} className={styles.link}>
```

Comment:
we shouldn't change it in the common table. we need to change the `getColumn` for episode function